projects/ngrx.io/content/guide/effects/index.md

```diff
@@ -36,8 +36,7 @@ The `Effect` decorator provides metadata to register observable side-effects in
 
 1.  Create an AuthEffects service that describes a source of login actions:
 
-```ts
-// ./effects/auth.effects.ts
+<code-example header="src/app/effects/auth.effects.ts">
```

Comment:
Remove the path from all the standalone examples, so its just `auth.effects.ts`, `app.module.ts`, etc .... If it were a complete example, we'd use the path.

Reply:
Like this?